models/queries.sql

```diff
@@ -2,27 +2,48 @@ WITH columns AS (
     SELECT
         column_name AS cn,
         table_catalog || '.' || table_schema || '.' || table_name AS tn
-    FROM "{{ var('source-database', target.database) }}".{{ source('information_schema', 'columns').include(database=false) }}
+
+    /*not working out of box so hardcoding bigquery requirments for looking at info-schema.
+    'region-us' looks at ALL of our BQ data-warehouse; we use 'schema' var based on needs.*/    
+    FROM `ele-prod-735126`.`region-us`.`INFORMATION_SCHEMA.COLUMNS`
+    --FROM "{{ var('source-database', target.database) }}".{{ source('information_schema', 'columns').include(database=false) }}
+    
     WHERE
+    
+        -- get id-columns of interest
         LOWER(column_name) IN {{ var('id-columns') }}
+
+        -- get tables of interest
+        {% if var('tables-to-include', undefined) %}
+        AND LOWER(table_name) IN {{ var('tables-to-include') }}
+        {% endif %}
+        
+        -- get schema(s) (BQ dataset) to include
         {% if var('schemas-to-include', undefined) %}
         AND LOWER(table_schema) IN {{ var('schemas-to-include') }}
         {% endif %}
-        {% if var('schemas-to-exclude', undefined) %}
+
+        -- not using schemas (exclude) for now
+        {# {% if var('schemas-to-exclude', undefined) %}
         AND NOT LOWER(table_schema) IN {{ var('schemas-to-exclude') }}
-        {% endif %}
-        {% if var('tables-to-include', undefined) %}
-        AND LOWER(table_name) IN {{ var('tables-to-include') }}
-        {% endif %}
-        {% if var('tables-to-exclude', undefined) %}
+        {% endif %} #}
+        
+        -- not using tables (exclude) for now
+        {# {% if var('tables-to-exclude', undefined) %}
         AND NOT LOWER(table_name) IN {{ var('tables-to-exclude') }}
-        {% endif %}
+        {% endif %} #}
```

Comment:
Since these default false if undefined you don't need to remove the if's. Leaving this closer to source makes it easier to update from upstream if we want to pull in more changes.